dll/win32/shell32/lang/pt-PT.rc

```diff
@@ -672,7 +679,7 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     ICON "", IDC_EDITTYPE_ICON, 5, 8, 20, 20
     EDITTEXT IDC_EDITTYPE_TEXT, 30, 10, 115, 14, ES_AUTOHSCROLL
-    PUSHBUTTON "Alterar &Icone...", IDC_EDITTYPE_CHANGE_ICON, 150, 10, 70, 14
+    PUSHBUTTON "Alterar &ícone...", IDC_EDITTYPE_CHANGE_ICON, 150, 10, 70, 14
```

Comment:
MS pt-PT picture proof for that accelerator please. The capital &I has been chosen here historically because &i does look suboptimal. But if MS does it like that, then I can live with it.

Reply:
No. "&i" is not a good option!

Reply:
Yes, I thought so. What does MS use instead?